tests/test_start_date.py

```diff
@@ -56,7 +58,14 @@ def test_run(self):
 
         # Select all streams and all fields within streams
         found_catalogs = menagerie.get_catalogs(conn_id)
-        incremental_streams = {key for key, value in self.expected_replication_method().items()
+        # removed 'abandoned_checkouts', as per the Doc:
+        #   https://help.shopify.com/en/manual/orders/abandoned-checkouts?st_source=admin&st_campaign=abandoned_checkouts_footer&utm_source=admin&utm_campaign=abandoned_checkouts_footer#review-your-abandoned-checkouts
+        # abandoned checkouts are saved in the Shopify admin for three months.
+        # Every Monday, abandoned checkouts that are older than three months are removed from your admin.
+        # Also no POST call is available for this endpoint: https://shopify.dev/api/admin-rest/2022-01/resources/abandoned-checkouts
+        expected_replication_method = self.expected_replication_method()
```

Comment:
Instead of popping this out from all tests, can we remove this from the base.py? Or update expected_replication_method function.

Reply:
We have removed from the `expected_streams` in the base file, but we cannot update `expected_replication_method` as it is used in discovery test.